requirements/base.txt

```diff
@@ -7,7 +7,7 @@ django-filter==0.13.0
 django-import-export==0.4.2
 django-mptt==0.8.7
 django-reversion==1.8.6
-django-rq==0.9.0
+django-rq>=1.2.0
```

Comment:
I would recommend to freeze the version to 1.2.0 to avoid random failures when django-rq or one of its dependencies becomes incompatible with Ralph. Since Ralph is not a library but an application, freezing dependencies does not hurt anyone.

Reply:
OK, will do. I chose those versions since at those releases the libraries became compatible with Python-3.7 (async is a keyword now). Do you happen to know whether it is possible to request those versions only in case of Python3.7?

Reply:
It's possible to do  requirements/py37.txt and requirements/py-other.txt but I would rather test everything with the same set of requirements than build more different builds of Ralph.

Reply:
OK, that's how we have it now.